### DISCUSSION TITLE
Ivan, are you alright?

### DISCUSSION THREAD

**Discussion by ShyLionTjmn** [vakata/jstree#2682] [General]
Hows your health?

**Comment by vakata** [comment]
Much better thank you - I have already started working on v.4 again - there will be some commits soon.

**Reply by chetanchopra26** [reply]
> Much better thank you - I have already started working on v.4 again - there will be some commits soon.

How's the v4 coming?

**Comment by ShyLionTjmn** [comment]
Ok, good to hear
